include/dca/util/containers/random_access_map_iterator.hpp

```diff
@@ -40,6 +40,13 @@ class RandomAccessMapIterator {
   template <class A, class B>
   using Conditional = std::conditional_t<is_const, A, B>;
 
+  // Provide STL iterator traits.
+  using iteator_category = std::bidirectional_iterator_tag;
```

Comment:
spelling

Reply:
thanks for noticing it. Fixed it. Let's see if the tests pass as nothing should have changed.